xwiki-pro-macros-ui/src/main/resources/XWiki/Macros/Panel.xml

```diff
@@ -212,7 +212,7 @@ This macro supports all Atlassian Confluence parameters as of version 7.9.</cont
 }
 .macro-border {
   margin-bottom: 10px;
-  overflow: hidden;
+  overflow: scroll;
```

Comment:
From my understanding, this means that browsers will now always display a scrollbar, I don't think that this is desired. See also [MDN on overflow](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow#values).